src/proc/proc.cpp

```diff
@@ -206,15 +198,9 @@ int feed_inputs(const contract_exec_args &args)
 
 int fetch_outputs(const contract_exec_args &args)
 {
-    if (read_contract_hp_outputs(args) != 0)
-    {
-        LOG_ERR << "Error reading HP output from the contract.";
-        return -1;
-    }
-
-    if (read_contract_fdmap_outputs(nplfds, args.nplbufs) != 0)
+    if (read_contract_hp_npl_outputs(args) != 0)
     {
-        LOG_ERR << "Error reading NPL output from the contract.";
+        LOG_ERR << "Error reading HP or NPL output from the contract.";
```

Comment:
Error message not needed. `read_contract_hp_npl_outputs` will log the correct error anyway.